gulpfile.babel.js

```diff
@@ -208,4 +233,9 @@ gulp.task('watch', gulp.series('default', function watch () {
     gulp.watch('_images/**/*.svg', gulp.series('svg'))
 
     gulp.watch('_styles/**/*.css', gulp.series('styles'))
+    gulp.watch('_scripts/**/*.js', gulp.series('scripts'))
```

Comment:
In a perfect world this would run webpack in watch mode so it's faster, but our script builds do not take that long so I'm ok with it.